### PR TITLE
[REF][report_purchase_order]Update design for the po report task#2035

### PR DESCRIPTION
**Pull request by IsabelEsparza**
@alan196

### CODE REVIEW

File: addons/report_purchase_order/__manifest__.py [hunk 1]
```diff
@@ -14,5 +14,6 @@
     ],
     'data': [
         'views/report_purchase_order.xml',
+        'views/report_papaerformat.xml',
```
Comment (alan196) [inline addons/report_purchase_order/__manifest__.py]:
```suggestion
        'views/report_paperformat.xml',
```

File: addons/report_purchase_order/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    from num2words import num2words
+except ImportError as err:
+    _logger.debug(err)
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order'
+
+
+    def _amount_to_text(self, amount, currency, partner_lang='es_MX'):
```
Comment (alan196) [inline addons/report_purchase_order/models/purchase.py:18]: please check this method inside Odoo

https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_currency.py#L99-L128

**Review by alan196 — CHANGES_REQUESTED** [review]

File: addons/report_purchase_order/models/purchase.py (hunk 2 above)
Comment (IsabelEsparza) [reply addons/report_purchase_order/models/purchase.py:18]: Only are for version 12 :crying_cat_face:

Comment (IsabelEsparza) [reply addons/report_purchase_order/models/purchase.py:18]: @alan196

**Pull request merged**